deploy/common/pmem-app-ephemeral.yaml

```diff
@@ -0,0 +1,22 @@
+# This demonstrates how to use inline pmem csi volumes
```

Comment:
Full sentence, end with a dot at the end?

Reply:
Fixed.